src/Components/Components.slnf

```diff
@@ -139,4 +140,4 @@
       "src\\WebEncoders\\src\\Microsoft.Extensions.WebEncoders.csproj"
     ]
   }
-}
+}
```

Comment:
This update was generated by VS. I think we not fight its view on trailing linebreaks or ordering.